CalibTracker/SiStripHitResolution/plugins/SiStripHitResolFromCalibTree.cc

```diff
@@ -0,0 +1,1752 @@
+//Original Author:  Christopher Edelmaier
+//        Created:  Feb. 11, 2010
+#include <memory>
+#include <string>
+#include <iostream>
+#include <fstream>
+#include <sstream>
+
+#include "CalibFormats/SiStripObjects/interface/SiStripDetCabling.h"
+#include "CalibFormats/SiStripObjects/interface/SiStripQuality.h"
+#include "CalibTracker/Records/interface/SiStripDetCablingRcd.h"
+#include "CalibTracker/Records/interface/SiStripQualityRcd.h"
+#include "CalibTracker/SiStripCommon/interface/SiStripDetInfoFileReader.h"
+//#include "CalibTracker/SiStripHitEfficiency/interface/TrajectoryAtInvalidHit.h"
+#include "CalibTracker/SiStripHitEfficiency/interface/SiStripHitEfficiencyHelpers.h"
+#include "CalibTracker/SiStripHitResolution/interface/HitResol.h"
+#include "CommonTools/ConditionDBWriter/interface/ConditionDBWriter.h"
+#include "CommonTools/TrackerMap/interface/TrackerMap.h"
+#include "CommonTools/UtilAlgos/interface/TFileService.h"
+#include "DataFormats/Common/interface/DetSetVector.h"
```

Comment:
It sems that these two includes can be removed (and probably also a few more)

Reply:
done in https://github.com/cms-sw/cmssw/pull/40754/commits/01bbe6e597150b216293e2dc33cdada65a192d3d.